File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/java/src/main/java/ro/altom/altunitytester/Commands/EnableLogging.java
package ro.altom.altunitytester.Commands;

import ro.altom.altunitytester.AltBaseSettings;

public class EnableLogging extends AltBaseCommand {
    public EnableLogging(AltBaseSettings altBaseSettings) {
        super(altBaseSettings);
    }
    public String Execute(){
        send(CreateCommand("enableLogging", altBaseSettings.logEnabled.toString()));
        String data = recvall();
        if (data.equals("OK")) {
            return data;
        }
        handleErrors(data);
        return "";
    }
}

File: Assets/AltUnityTester/AltUnityDriver/Commands/EnableLogging.cs
public class EnableLogging : AltBaseCommand
{
    public EnableLogging(SocketSettings socketSettings) : base(socketSettings)
    {
    }
    public void Execute(){
        Socket.Client.Send(toBytes(CreateCommand("enableLogging", SocketSettings.logFlag.ToString())));
        var data = Recvall();
        if (data.Equals("Ok"))
            return;
        HandleErrors(data);
    }
}
